src/fshaper/fshaper.sh

```diff
@@ -4,12 +4,10 @@
 # License GPL v2
 
 TCFILE=""
-if [ -x "/usr/sbin/tc" ]; then
-	TCFILE="/usr/sbin/tc"
-elif [ -x "/sbin/tc" ]; then
-	TCFILE="/sbin/tc";
+if [ -x "$(PATH=/usr/sbin:/sbin:/run/current-system/sw/bin command -v tc)" ]; then
+	TCFILE="$(PATH=/usr/sbin:/sbin:/run/current-system/sw/bin command -v tc)"
```

Comment:
Double call to command.

Assigne it to TCFILE, then check TCFILE

Reply:
I've reworked the test. Thanks for reviewing!